build_projects/dotnet-host-build/PackageTargets.cs

```diff
@@ -13,6 +13,12 @@ namespace Microsoft.DotNet.Host.Build
 {
     public static class PackageTargets
     {
+        public static readonly string[] ProjectsToPack  = new string[]
+        {
+            "Microsoft.DotNet.PlatformAbstractions",            
```

Comment:
Should this be named "Microsoft.Extensions.PlatformAbstractions" to fit better with "Microsoft.Extensions.DependencyModel"?